### ISSUE TITLE
[SCREENREADER, KEYBOARD, COGNITION]: Links SHOULD look like links

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#6507]
**Feedback framework**
```diff
- **MUST** for if the feedback must be applied 
! **SHOULD** if the feedback is best practice 
+ **CONSIDER** for suggestions/enhancements 
``` 

## Description

```diff
! Links **SHOULD** look and behave like links.
```
See screenshots below. While testing with screenreaders, the items listed under **Get VA disability compensation (pay)** are confusing. I thought I could stop the screen reader by tapping on the text that didn't look like it was part of the link. It was confusing that there was a pointer on what looked like non-linked text. I was surprised to find that all of the text was a link, as I went to the linked content. This text that does not look like a link is not compliant with Section 508 as outlined here in the WCAG 2.0 success criteria:

WCAG 2.0 has 2 additional requirements for body text links that are not underlined by default:
- The link text must have a 3:1 contrast ratio from the surrounding non-link text.
- The link must present a "non-color designator" (typically the introduction of the underline) on both mouse hover and keyboard focus.

These two requirements help ensure that all users can differentiate links from non-link text, even if they have low vision, color deficiency, or have overridden page colors.

**Note:** I am not sure if this is a CMS issue or for those entering content into the CMS. If this is a CMS issue, please comment on this ticket and I will reassign accordingly.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

- As a keyboard user, I want to confidently click on the page to activate links.
- As a sighted screenreader user, I want to confidently click non-link test on the page to stop the screenreader.

## Environment

* Operating System: any
* Browser: any
* Screenreading device: any
* Server destination: production

## Steps to Recreate

1. Enter `https://www.va.gov/disability/` in browser
2. Start screenreading device 
3. Navigate to the list items under **Get VA disability compensation (pay)**
4. Attempt to stop the screenreader reading by clicking the non-link text
5. Validate that what looks like regular text is linked

## Possible Fixes (optional)

- Possibly end the link after the right-angle bracket so it is clear what is the link and it is concise.
- **Enhancement:** Consider adding `role="presentation"` and `aria-hidden="true"` to the angle bracket so that it doesn't read out to screenreaders.

```diff
! <!-- current text sample -->
! <li class="hub-page-link-list__item">
!     <a href="/disability/eligibility" id="7ab0783f3f0bf783dc41fa8322fbedc6">
!             <span class="hub-page-link-list__header">Eligibility</span>
!                 <img class="all-link-arrow" src="https://prod-va-gov-assets.s3-us-gov-west-1.amazonaws.com/img/arrow-right-blue.svg" alt="right-arrow">
!         
!             <p class="va-nav-linkslist-description">Find out if you’re eligible for VA disability benefits for a presumptive disability or other service-connected condition.</p>
!     </a>
! </li>

<!-- possible fix -->
<li class="hub-page-link-list__item">
    <a href="/disability/eligibility" id="7ab0783f3f0bf783dc41fa8322fbedc6">
            <span class="hub-page-link-list__header">Eligibility</span>
                <img class="all-link-arrow" src="https://prod-va-gov-assets.s3-us-gov-west-1.amazonaws.com/img/arrow-right-blue.svg" alt="right-arrow" 
+ role="presentation" aria-hidden="true">
+      </a>      
+            <p class="va-nav-linkslist-description">Find out if you’re eligible for VA disability benefits for a presumptive disability or other service-connected condition.</p>
</li>
```

## WCAG or Vendor Guidance (optional)

* [WebAIM Links and hypertext](https://webaim.org/techniques/hypertext/link_text)
* [WCAG 2.0 - Failure of Success Criterion 1.4.1 due to creating links that are not visually evident without color vision](https://www.w3.org/TR/WCAG20-TECHS/F73.html)
* [HTML5 Accessibility: aria-hidden and role=”presentation”](http://john.foliot.ca/aria-hidden/)

## Screenshots or Trace Logs

![vsa-pw-links-and-p1](https://user-images.githubusercontent.com/57469/75911098-0047f600-5e1d-11ea-9313-80b8446bf899.png)

![vsa-pw-links-and-p2](https://user-images.githubusercontent.com/57469/75911109-06d66d80-5e1d-11ea-929c-aaf2c96fb4eb.png)

**Comment by kelsonic** [comment]
tagging for awareness, not really sure how actionable this ticket is CC @brianalloyd

**Comment by laflannery** [comment]
This has been resolved. Only the title text is linked now rather than the paragraph text below. Will close in 1 week unless told otherwise
